static/src/js/containers/DownloadHistoryContainer/__tests__/DownloadHistoryContainer.test.js

```diff
@@ -1,89 +1,250 @@
-        onDeleteRetrieval: jest.fn()
+  describe('when deleting a retrieval', () => {
+    const initialHistory = [{
+      id: '8069076',
+      jsondata: {},
+      created_at: '2019-08-25T11:58:14.390Z',
+      collections: [{}]
+    }]
+
+    beforeEach(() => {
+      nock(/.*/)
+        .get(/retrievals/)
+        .reply(200, initialHistory)
+    })
+
+    test('handles successful deletion', async () => {
+      nock(/.*/)
+        .delete(/retrievals\/8069076/)
+        .reply(200, {})
+
+      render(
+        <Provider store={store}>
+          <MemoryRouter>
+            <DownloadHistoryContainer />
+          </MemoryRouter>
+        </Provider>
+      )
+
+      await waitFor(() => {
+        expect(lastPassedOnDeleteRetrieval).toEqual(expect.any(Function))
+      })
+
+      await act(async () => {
+        if (!lastPassedOnDeleteRetrieval) {
+          throw new Error('onDeleteRetrieval callback was not captured by mock')
+        }
+
+        await lastPassedOnDeleteRetrieval('8069076')
+      })
+
+      await waitFor(() => {
+        expect(addToast).toHaveBeenCalledWith('Retrieval removed', {
+          appearance: 'success',
+          autoDismiss: true
+        })
+      })
+
+      expect(handleError).not.toHaveBeenCalled()
+
+      await waitFor(() => {
+        expect(DownloadHistory).toHaveBeenLastCalledWith(
+          expect.objectContaining({
+            retrievalHistory: []
+          }),
+          expect.anything()
+        )
+      })
+    })
+
+    test('handles error when deletion fails', async () => {
+      nock(/.*/)
+        .delete(/retrievals\/8069076/)
+        .replyWithError('Failed to delete retrieval')
+
+      render(
+        <Provider store={store}>
+          <MemoryRouter>
+            <DownloadHistoryContainer />
+          </MemoryRouter>
+        </Provider>
+      )
+
+      await waitFor(() => {
+        expect(lastPassedOnDeleteRetrieval).toEqual(expect.any(Function))
+      })
+
+      await act(async () => {
+        if (!lastPassedOnDeleteRetrieval) {
+          throw new Error('onDeleteRetrieval callback was not captured by mock')
+        }
+
+        await lastPassedOnDeleteRetrieval('8069076')
       })
 
-      expect(enzymeWrapper.find(DownloadHistory).length).toBe(1)
+      await waitFor(() => {
+        expect(handleError).toHaveBeenCalledWith({
+          error: expect.any(Error),
+          action: 'handleDeleteRetrieval',
+          resource: 'retrieval',
+          verb: 'deleting',
+          notificationType: 'banner'
+        })
+      })
+
+      expect(addToast).not.toHaveBeenCalledWith('Retrieval removed', expect.anything())
+
+      await waitFor(() => {
+        expect(DownloadHistory).toHaveBeenLastCalledWith(
+          expect.objectContaining({
+            retrievalHistory: initialHistory
+          }),
+          expect.anything()
+        )
+      })
+    })
+  })
+
+  describe('mapStateToProps', () => {
+    test('maps state to props correctly', () => {
```

Comment:
This is not testing `mapStateToProps` from the component, you have to import it to test it. 

Also missing tests for `mapDispatchToProps`